dist/profile/manifests/updates.pp

```diff
@@ -0,0 +1,23 @@
+#
+# Defines a updates.jenkins.io server for serving updatesite, aims to deal with volume management
+#
+class profile::updates() {
+  include ::stdlib
+  # volume configuration is in hiera
+  include ::lvm
+
+
+  if str2bool($::vagrant) {
+    # during serverspec test, fake /dev/oracleoci/oraclevdb by a loopback device
+    exec { 'create /tmp/oraclevdb':
+      command => 'dd if=/dev/zero of=/tmp/oraclevdb bs=1M count=16; losetup /dev/loop0; losetup /dev/loop0 /tmp/oraclevdb',
+      unless  => 'test -f /tmp/oraclevdb',
+      path    => '/usr/bin:/usr/sbin:/bin:/sbin',
+      before  => Physical_volume['/dev/loop0'],
+    }
+  }
```

Comment:
```suggestion
```

Once https://github.com/jenkins-infra/jenkins-infra/pull/2290 is approved and merged, this code block is not required anymore.

Sounds like this profile might even be removed (not needed anymore as the profile `vagrant` manage automatic LVM if you specified thourhg hiera)